src/govee_local_api/__init__.py

```diff
@@ -5,4 +5,4 @@
 
 __all__ = ["GoveeController", "GoveeDevice", "GoveeLightCapability"]
 
-__version__ = "1.4.5"
+__version__ = "1.4.6"
```

Comment:
The bump is not required, this is already the next version

Reply:
done